init.go

```diff
@@ -73,14 +80,18 @@ func treatmentServiceDefinition(sv *ecs.Service) *ecs.Service {
 	sv.Deployments = nil
 	sv.Events = nil
 	sv.PendingCount = nil
-	sv.PropagateTags = nil
 	sv.RunningCount = nil
 	sv.Status = nil
 	sv.TaskDefinition = nil
 	sv.TaskSets = nil
 	sv.ServiceArn = nil
 	sv.RoleArn = nil
 	sv.ServiceName = nil
+
+	if *sv.PropagateTags != "SERVICE" && *sv.PropagateTags != "TASK_DEFINITION" {
```

Comment:
If not set, it will be `NONE`, which is an invalid value in create...
```
"propagateTags": "NONE",
```

error:
```
2021/04/09 10:33:26 create FAILED. failed to create service: InvalidParameterException: Propagate tags should be one of [SERVICE,TASK_DEFINITION]
```

Reply:
OK, DescribeServices returns "NONE", but CreateService API doesn't accept "NONE"... 😭 

https://docs.aws.amazon.com/ja_jp/AmazonECS/latest/APIReference/API_CreateService.html#API_CreateService_RequestSyntax